src/net_processing.cpp

```diff
@@ -2370,6 +2370,28 @@ static void ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv, const CChainPar
     connman.PushMessage(&peer, std::move(msg));
 }
 
+/**
+ * Announce transactions a peer is missing after reconciliation is done.
+ * No need to add transactions to peer's filter or do checks
+ * because it was already done when adding to the reconciliation set.
+ */
+void static AnnounceTxs(std::vector<uint256> remote_missing_wtxids, CNode* pto, CNetMsgMaker msgMaker, CConnman* connman)
+{
+    if (remote_missing_wtxids.size() != 0) {
+        std::vector<CInv> remote_missing_invs;
+        for (uint256 wtxid: remote_missing_wtxids) {
```

Comment:
nit: I think this could be a const reference as well, and probably there's some modern c++ you could write so that we're not making extra copies in this loop, someone else can chime in and explain how to do that...